app/config.js

```diff
@@ -23,7 +23,7 @@ window.OPENSHIFT_CONFIG = {
   },
   additionalServers: [{
     protocol: "https",
-    hostPort: "apiserver-svc-cat.10.0.0.25.xip.io:6443",
+    hostPort: "apiserver-svc-cat.10.0.0.25.xip.io",
```

Comment:
I don't believe there is a good way to see the error. Same as in console. The discovery is done during the angular bootstrap process and if it fails the page is redirected so any failure message is not retrievable.